pydash/predicates.py

```diff
@@ -77,6 +78,34 @@
 RegExp = type(re.compile(''))
 
 
+def eq(value, other):
+    """Checks if :attr:`value` is equal to :attr:`other`.
+
+    Args:
+        value (mixed): Value to compare.
+        other (mixed): Other value to compare.
+
+    Returns:
+        bool: Whether :attr:`value` is equal to :attr:`other`.
+
+    Example:
+
+        >>> eq({'a': 1}, {'a': 1})
+        True
+        >>> eq(None, None)
+        True
+        >>> eq(None, '')
+        False
+        >>> eq('a', 'a')
+        True
+        >>> eq(1, str(1))
+        False
+
+    .. versionadded:: TODO
+    """
+    return is_equal(value, other)
```

Comment:
JavaScript '===' is the same as 'is' in Python. So 'eq' should check whether the two objects being compared are the same object.